viewer/.eslintrc.js

```diff
@@ -22,7 +22,7 @@ module.exports = {
     // This will display prettier errors as ESLint errors. Make sure this is always the last configuration in the extends array.
     'plugin:prettier/recommended'
   ],
-  ignorePatterns: ['.eslintrc.js', '**/dist/**/*', '**/*/draco_decoder_gltf.js'],
+  ignorePatterns: ['.eslintrc.js', '**/dist/**/*', '**/*/draco_decoder_gltf.js', '**/wasm/**/*.js'],
```

Comment:
Is this supposed to be part of this PR?

Reply:
@haakonflatval-cognite: I decided to leave it in. The reasons was that there was a lint failure in CI, so I used `yarn lint` to fix it. I then noticed a bunch of warnings in the generated WASM wrappers. I mentioned this in the PR description :)